### ISSUE TITLE
CHM Documentation is very outdated

### ISSUE THREAD

**Issue by glmnet** [NModbus/NModbus#10]
I believe this should be updated, removed or at least clearly state this in readme, as its misleading information.

**Comment by rquackenbush** [comment]
Good point. I’ll ditch this file and look at generating more up-to-date docs.

**Comment by FabioGenesy** [comment]
I installed the library with Nuget (version 1.11). How I can find chm documentation?

**Comment by rquackenbush** [comment]
Hi - we’re in the process of transitioning to a newer / better
documentation format. Please find the new version here:

https://nmodbus.github.io/api/NModbus.html
